src/unclogged/core.clj

```diff
@@ -29,6 +29,26 @@
     "3164" MessageFormat/RFC_3164
     "5424" MessageFormat/RFC_5424))
 
+(defn ^:private ->syslog-msg
+  "Turns defaults + message map into a SyslogMessage."
+  [defaults message]
+  (let [{:keys [message
+                message-id
+                app-name
+                hostname
+                process-id
+                facility
+                severity]}
+        (merge defaults message)]
+    (doto (SyslogMessage.)
```

Comment:
let's say you have a java type that is literally just a pile of getters and setters, but because java's excellent type system you actually need an instance of that type; however on the clojure side you obviously want a map, because screw all of that noise. `(proxy FancyClass (getTransmodulator [this x y z] (:transmodulator my-map)))`